css/datagrid/datagrid.css

```diff
@@ -83,42 +83,160 @@
     --highcharts-highlight-color-10: rgb(58, 63, 71);
 }
 
-/* stylelint-disable max-line-length */
+.hcdg-theme-default {
+    --border-radius: 5px;
+    --color: var(--highcharts-neutral-color-100);
+    --padding: 8px;
+    --border-width: 1px;
+    --border-color: var(--highcharts-neutral-color-20);
+    --header-font-weight: bold;
+    --background-color: var(--highcharts-background-color);
+    --row-even-background-color: var(--highcharts-neutral-color-3);
+    --header-row-background-color: var(--highcharts-neutral-color-5);
 
-/* ==== DataGrid Core ==== */
-.highcharts-datagrid-container * {
-    box-sizing: border-box;
-    font-family:
-        -apple-system,
-        BlinkMacSystemFont,
-        "Segoe UI",
-        Roboto,
-        Helvetica,
-        Arial,
-        "Apple Color Emoji",
-        "Segoe UI Emoji",
-        "Segoe UI Symbol",
-        sans-serif;
+    --row-hovered-background-color: var(--highcharts-highlight-color-10);
+    --column-hovered-background-color: rgba(0, 0, 255, 0.03);
+    --header-column-hovered-background-color: rgba(0, 0, 255, 0.07);
+
+    --row-synced-background-color: rgba(255, 0, 0, 0.1);
+    --column-synced-background-color: rgba(255, 0, 0, 0.06);
+    --header-column-synced-background-color: rgba(255, 0, 0, 0.1);
+
+    --header-text-align: center;
+
+    --credits-font-size: 0.6em;
+    --credits-text-align: right;
```

Comment:
I have provided suggested fix for colors in light/dark mode to @sebastianbochan. Let's not assume what background-color people use in their dark modes, and override that by setting our orn. That means that container is transparent, and users just need to make sure that they have correct color on parent elements or body.

Wo do, though, set `#ffffff` or `#000000` on text and table-borders just so that they see **something** if starting with no theme. Basically to avoid confusion.

Regarding prefix: IMO we should keep these as short and simple as possible. If they already have e.g.` --color `defined in `:root` or somewhere else we always do some resets in `.highcharts-datagrid-container` so should be safe.

Reply:
So what is our conclusion, do we add any prefixes to public variables?